src-plugins/manualRegistration/manualRegistrationLandmark.cpp

```diff
@@ -99,7 +99,7 @@ manualRegistrationLandmark::~manualRegistrationLandmark()
     HandleWidget->RemoveAllObservers();
     HandleWidget->Delete();
     HandleWidget=0;
-    View->RemoveObserver(SeedCallback);
```

Comment:
I wanted to avoid other bugs using exhaustive delete/clean functions which were already coded :)